doc/tutorials/duckdb-native-sqlalchemy.md

```diff
@@ -128,6 +128,7 @@ To suppress it, add this at the top of your notebook/script:
 ```{code-cell} ipython3
 from sql.warnings import JupySQLDataFramePerformanceWarning
 import warnings
+
```

Comment:
Based on this [comment](https://github.com/ploomber/jupysql/issues/728#issuecomment-1652351592) it looks like we need to add a boxplot example in this guide